src/main/java/uk/co/ramp/covid/simulation/Model.java

```diff
@@ -224,14 +227,18 @@ public boolean isValid() {
             p.setExternalInfectionDays(externalInfectionDays);
             p.seedVirus(nInitialInfections);
 
-            if (!lockdownComponents.isEmpty()) {
-                LockdownComponentFactory f = new LockdownComponentFactory();
-                List<LockdownComponent> lockdowns = f.buildComponents(lockdownComponents, p);
-                for (LockdownComponent c : lockdowns) {
+            // Backwards compatability
```

Comment:
I'd prefer not to unnecessarily have two ways of doing the same thing.  It's not like we have a large user base with lots of old .json files that we need to support.

Reply:
100% agree - this is all earmarked to go once we confirm the components we are going to use with Paul. As is the old parsing code (forgetting to remove that in the commit was actually a mistake so thanks very much for pointing that out!).

I'll probably close this PR once Paul has seen it and rebase all this before we push it into develop to avoid all these extra "trying stuff out" commits.